### PR TITLE
Add Code Owners for Tasks

### PR DESCRIPTION
**Pull request by vincentdass**

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,257 @@
...
+Tasks/AzureAppServiceManageV0/*     @kmkumaran      @SumiranAgg     @vincentdass
+
+Tasks/AzureCLIV1/*      @sachinma
+
+Tasks/AzureCloudPowerShellDeploymentV1/*    @kmkumaran
+
+Tasks/AzureFileCopyV1/*    @kmkumaran
+
+Tasks/AzureFileCopyV2/*    @kmkumaran
+
```
Comment (kmkumaran) [inline .github/CODEOWNERS]: The new Azure function and container tasks that we added are missing in the list.

Comment (vincentdass) [reply .github/CODEOWNERS]: added missing task. thanks.

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -0,0 +1,267 @@
...
+Tasks/DownloadPipelineArtifactV0/*      @mihaif
+
+Tasks/DownloadSecureFileV1/*    @madhurig
+
+Tasks/ExtractFilesV1/*      @bryanmacfarlane
+
+Tasks/FtpUploadV1/*     @madhurig
+
+Tasks/GithubReleaseV0/*     @prativen
```
Comment (prativen) [inline .github/CODEOWNERS]: prativen and mdmdakbar

File: .github/CODEOWNERS [hunk 3]
```diff
@@ -0,0 +1,267 @@
...
+
+Tasks/AzureFileCopyV3/*    @kmkumaran
+
+Tasks/AzureFunctionDeploymentV1/*    @kmkumaran      @SumiranAgg     @vincentdass
+
+Tasks/AzureFunctionOnContainerDeploymentV1/*    @kmkumaran      @SumiranAgg     @vincentdass
+
+Tasks/AzureFunctionV1/*     @bansalaseem
+
+Tasks/AzureIoTEdgeV2/*      @prativen
```
Comment (prativen) [inline .github/CODEOWNERS]: add michaeljqzq for now. Will include Shyama's as well

File: .github/CODEOWNERS [hunk 4]
```diff
@@ -0,0 +1,267 @@
...
+
+Tasks/AzureKeyVaultV1/*     @bansalaseem
+
+Tasks/AzureMonitorAlertsV0/*    @kmkumaran
+
+Tasks/AzureMonitorV0/*      @bansalaseem
+
+Tasks/AzureMysqlDeploymentV1/*  @kmkumaran
```
Comment (RoshanKumarMicrosoft) [inline .github/CODEOWNERS]: Add rok

File: .github/CODEOWNERS [hunk 5]
```diff
@@ -0,0 +1,267 @@
...
+
+Tasks/ManualInterventionV8/*
+
+Tasks/MavenV2/*     @madhurig
+
+Tasks/MavenV3/*     @madhurig
+
+Tasks/MSBuildV1/*   @daveken
+
+Tasks/MysqlDeploymentOnMachineGroupV1/*     @kmkumaran
```
Comment (RoshanKumarMicrosoft) [inline .github/CODEOWNERS]: Add rok

File: .github/CODEOWNERS [hunk 6]
```diff
@@ -0,0 +1,267 @@
...
+
+Tasks/DockerV1/*    @bansalaseem
+
+Tasks/DockerComposeV0/*     @bansalaseem
+
+Tasks/DotNetCoreCLIV2/*     @sachinma
+
+Tasks/DotNetCoreInstallerV0/*   @sachinma
+
+Tasks/DownloadBuildArtifactsV0/*    @Lovakumar
```
Comment (Lovakumar) [inline .github/CODEOWNERS]: Add Omesh for DownloadBuildArtifactsV0

File: .github/CODEOWNERS [hunk 7]
```diff
@@ -0,0 +1,267 @@
...
+
+Tasks/DownloadFileshareArtifactsV0/*    @Lovakumar
```
Comment (Lovakumar) [inline .github/CODEOWNERS]: Add Omesh for DownloadFileshareArtifactsV0

File: .github/CODEOWNERS [hunk 8]
```diff
@@ -0,0 +1,267 @@
...
+
+Tasks/InstallAppleProvisioningProfileV1/*   @madhurig
+
+Tasks/InstallSSHKeyV0/*     @madhurig
+
+Tasks/InvokeRestApiV1/*     @bansalaseem
+
+Tasks/JavaToolInstallerV0/*     @madhurig
+
+Tasks/JenkinsDownloadArtifactsV1/*      @Lovakumar
```
Comment (Lovakumar) [inline .github/CODEOWNERS]: Add Kalyan for JenkinsDownloadArtifactsV1

File: .github/CODEOWNERS [hunk 9]
```diff
@@ -0,0 +1,267 @@
...
+
+Tasks/JenkinsDownloadArtifactsV1/*      @Lovakumar      @kasubram
+
+Tasks/JenkinsQueueJobV2/*   @madhurig
+
+Tasks/KubernetesV0/*    @bansalaseem
+
+Tasks/KubernetesV1/*    @bansalaseem
+
+Tasks/ManualInterventionV8/*
```
Comment (bansalaseem) [inline .github/CODEOWNERS]: you can mark this as bansalaseem

File: .github/CODEOWNERS [hunk 10]
```diff
@@ -0,0 +1,267 @@
...
+
+Tasks/QueryWorkItemsV0/*    @bansalaseem
+
+Tasks/QuickPerfTestV1/*     @smalpani
+
+Tasks/RunDistributedTestsV1/*   @smalpani
+
+Tasks/RunJMeterLoadTestV1/*     @smalpani
+
+Tasks/RunLoadTestV1/*   @smalpani
```
Comment (cltshivash) [inline .github/CODEOWNERS]: assign to @cltshivash

File: .github/CODEOWNERS [hunk 11]
```diff
@@ -0,0 +1,267 @@
...
+
+Tasks/UsePythonVersionV0/*      @madhurig
+
+Tasks/UseRubyVersionV0/*    @hross
+
+Tasks/VSBuildV1/*   @daveken
+
+Tasks/VsTestV1/*    @smalpani
+
+Tasks/VsTestV2/*    @smalpani
```
Comment (cltshivash) [inline .github/CODEOWNERS]: assign to @cltshivash

File: .github/CODEOWNERS [hunk 12]
```diff
@@ -0,0 +1,267 @@
...
+
+Tasks/VsTestPlatformToolInstallerV1/*   @smalpani
```
Comment (cltshivash) [inline .github/CODEOWNERS]: assign to @cltshivash

File: .github/CODEOWNERS [hunk 13]
```diff
@@ -0,0 +1,267 @@
...
+
+Tasks/PowerShellOnTargetMachinesV3/*    @kmkumaran
+
+Tasks/PublishBuildArtifactsV1/*     @madhurig
+
+Tasks/PublishCodeCoverageResultsV1/*    @madhurig
+
+Tasks/PublishPipelineArtifactV0/*       @mihaif
+
+Tasks/PublishSymbolsV2/*    @smalpani
```
Comment (cltshivash) [inline .github/CODEOWNERS]: PublishSymbols assignment is incorrect.

File: .github/CODEOWNERS [hunk 14]
```diff
@@ -0,0 +1,267 @@
...
+
+Tasks/PublishTestResultsV1/*    @smalpani
+
+Tasks/PublishTestResultsV2/*    @smalpani
```
Comment (cltshivash) [inline .github/CODEOWNERS]: Assign to sada (@SadagopanRajaram)

File: .github/CODEOWNERS [hunk 15]
```diff
@@ -0,0 +1,267 @@
...
+
+Tasks/CUrlUploaderV2/*      @madhurig
+
+Tasks/DecryptFileV1/*   @madhurig
+
+Tasks/DelayV1/*     @bansalaseem
+
+Tasks/DeleteFilesV1/*   @bryanmacfarlane
+
+Tasks/DeployVisualStudioTestAgentV2/*       @smalpani
```
Comment (cltshivash) [inline .github/CODEOWNERS]: assign to @Cltshivash

Comment (cltshivash) [inline .github/CODEOWNERS]: assign to @cltshivash

File: .github/CODEOWNERS (hunk 13 above)
Comment (vincentdass) [reply .github/CODEOWNERS]: who is the task owner?

Comment (kmkumaran) [reply .github/CODEOWNERS]: @arunkm - I am marking you as the owner based on history in commits. Please feel free to call out if you are not the owner.

File: .github/CODEOWNERS [hunk 16]
```diff
@@ -0,0 +1,267 @@
+Tasks/AndroidSigningV2/*    @madhurig
+
+Tasks/AndroidSigningV3/*    @madhurig
+
+Tasks/ANTV1/*   @madhurig
+
+Tasks/AppCenterDistributeV1/*   @madhurig
```
[the remark this answers is not in the document]
Comment (kmkumaran) [reply .github/CODEOWNERS]: @DennisPan - I am updating you as the owner. Feel free to get back if you need this to be changed.

**Pull request merged**